django/db/models/manager.py

```diff
@@ -23,15 +22,12 @@ def ensure_default_manager(cls):
         setattr(cls, 'objects', SwappedManagerDescriptor(cls))
         return
     if not getattr(cls, '_default_manager', None):
-        # Create the default manager, if needed.
-        try:
-            cls._meta.get_field('objects')
+        if any(f.name == 'objects' for f in cls._meta.fields):
```

Comment:
Good question: get_fields() necessarily warms the relation tree cache for everyone, we don’t want this to happen before the apps registry is ready.